### DISCUSSION TITLE
波茲曼entropy公式真偉大！

### DISCUSSION THREAD

**Discussion by LeoTsai999** [1101-GS4719-C/Python-Community-GS4719-C#2141] [General]
波茲曼提出了entropy formula，連結系統組態數與entropy的關聯。我覺得這是一個非常重要的貢獻。
這件事啟發了我對於理論研究的想法。我認為一個理論物理學家，應該要像波茲曼一樣從理論中預測某些目前實驗上還未證實的事情，給予科學研究一個方向！

**Comment by zhilin93517** [comment]
我不太清楚這些理論是如何推導的，可以請版主提出一點例子嗎？這樣可以更有幫助我對於這方面的理解，我相信這會對我未來研究複雜系統更有幫助

**Comment by seastar0687** [comment]
波茲曼做出了十分重要的貢獻，他可以說為現在熱力學研究打下了基礎，但我們不能忘記在熱力學課本前面提醒我們的波茲曼還有他的學生也步上的末路，或許我們在研究之餘，也要好好的重視心理健康，才能破除這熱力學魔咒。
